lib/benchee/benchmark/runner.ex

```diff
@@ -4,10 +4,11 @@ defmodule Benchee.Benchmark.Runner do
   run time and memory usage to each scenario.
   """
 
-  alias Benchee.{Benchmark, MemoryMeasure}
+  alias Benchee.Benchmark
   alias Benchee.Benchmark.{Scenario, ScenarioContext}
   alias Benchee.Utility.{RepeatN, Parallel}
   alias Benchee.Configuration
+  alias Benchee.Benchmark.Measurer
```

Comment:
Maybe place this alias with the others from Benchmark

Reply:
ah damn me, totally - thanks Eric!